internal/charmstore/search_test.go

```diff
@@ -77,6 +76,8 @@ func (s *StoreSearchSuite) TestSuccessfulExport(c *gc.C) {
 		readACLs := []string{params.Everyone}
 		if r.User != "" {
```

Comment:
I wanted to reduce the amount of changes to existing tests, this way they could be left alone in many places. Also it gives a nice way to specify which ones should be promulgated. exportTestCharms is used in a lot of places.